package.json

```diff
@@ -84,6 +84,12 @@
                 "category": "Yang Diagram"
             }
         ],
+        "jsonValidation": [
+            {
+                "fileMatch": "yang.settings",
+                "url": "https://raw.githubusercontent.com/TypeFox/yang-lsp/master/schema/yang-lsp-settings-schema.json"
```

Comment:
@dhuebner `master` could now be changed to `v0.7.6` when yang-vscode starts to use that version, and so on

Reply:
@esmasth 
Yes, I created v0.7.6 tag for that.
I need to adjust the releasing notes too, so we don't forget to update the version next time.